pyseir/rt/infer_rt.py

```diff
@@ -64,14 +73,15 @@ def _generate_input_data(
         include_deaths=include_deaths,
         figure_collector=figure_collector,
         display_name=fips,
+        log=rt_log.new(fips=fips),
     )
     return df
 
 
 def filter_and_smooth_input_data(
-    df: [pd.DataFrame], display_name, include_deaths=False, figure_collector=None
+    df: [pd.DataFrame], display_name, include_deaths, figure_collector, log
```

Comment:
can you add types to the rest of the variables?  Also, is df `pd.DataFrame` or truly a list?

Reply:
Done. And df is more generally a dict-like object with at least the keys "cases" and "deaths".